scripts/update-neuron-compatible.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs')
+const path = require('path')
+
+const exec = () => {
+  const compatiblePath = path.resolve(__dirname, '../compatible.json')
+  const info = require(compatiblePath)
+  const packagePath = path.resolve(__dirname, '../package.json')
+  const currentNeuronFullVersion = require(packagePath).version
```

Comment:
`fullVersion` is confusing because `fullVersion` in `json` points to `ckb full node`.